### PR TITLE
add badges to readme

### PR DESCRIPTION
**Pull request by davidak**
I think the logo in front of the heading don't look good. That might be a reason why no body does it.

Rendered view: https://github.com/greenbone/gvm-tools/blob/627d28896ffa4a28fcfb374008a5055ff1a60b59/README.md

What do you think @wiegandm?

We can improve the text in the next step.

### CODE REVIEW

**Comment by bjoernricks** [comment]
Looks really nice :+1:

**Comment by wiegandm** [comment]
> I think the logo in front of the heading don't look good.

You are right, it did look better in my head ...

I have no objections to removing the logo for now, unless you have an idea for 
a better place.

**Comment by davidak** [comment]
First version: small logo in front of heading

screenshot from 2018-09-25 17-22-45

Second version: just heading, no logo

screenshot from 2018-09-25 17-27-01

Third version: logo with text above heading

screenshot from 2018-09-25 17-42-49

Fourth version: logo, heading and badges centered

screenshot from 2018-09-25 17-56-04

What do you think?

File: README.md [hunk 1]
```diff
@@ -1,10 +1,23 @@
-[![image](https://circleci.com/gh/greenbone/gvm-tools/tree/master.svg?style=svg)](https://circleci.com/gh/greenbone/gvm-tools/tree/master)
+<div align="center">
```
Comment (bjoernricks) [inline README.md]: align="center" is really from the 90ies. You could try
```html
<div style="display:flex; justify-content: center">
```
instead

Comment (davidak) [reply README.md]: Thanks

**Comment by davidak** [comment]
@janowagner  said the third version is the best tradeoff. We will show it also to marketing and ask for their opinion.

Ready to merge for now.

**Pull request merged**